ts/ui/menu/Menu.ts

```diff
@@ -402,13 +402,13 @@ export class Menu {
         this.variable<boolean>('semantics'),
         this.variable<string> ('zoom'),
         this.variable<string> ('zscale'),
-        this.variable<string> ('renderer', (jax: string) => this.setRenderer(jax)),
+        this.variable<string> ('renderer', (jax) => this.setRenderer(jax)),
```

Comment:
There should be no need for explicit lambda expressions.
But at least we can get rid of paranthese